### DISCUSSION TITLE
Module: Foca or Foca-like functionality

### DISCUSSION THREAD

**Discussion by nicpenning** [blacklanternsecurity/bbot#747] [Ideas]
"FOCA is a tool used mainly to find metadata and hidden information in the documents it scans. These documents may be on web pages, and can be downloaded and analysed with FOCA."

Link to the project: https://github.com/ElevenPaths/FOCA/tree/master

Hard to say what the best approach is to implement so it's just an idea :) and maybe some of these features are taken care of in other modules.

**Comment by TheTechromancer** [comment]
Good idea! This is a feature a lot of us have been wanting. There is already a discussion here:

https://github.com/blacklanternsecurity/bbot/discussions/717.
